### ISSUE TITLE
HALFor is not included in module __init__.py

### ISSUE THREAD

**Issue by xeno-john** [jtc42/fastapi-hypermodel#5]
The new HALFor class cannot be used because it is not included in the __init__ file of the module.

**Comment by xeno-john** [comment]
This is the file:
```
from .hypermodel import HyperModel, UrlFor, LinkSet

__all__ = ["HyperModel", "UrlFor", "LinkSet"]

```

**Comment by jtc42** [comment]
Thanks for spotting this. `HALFor` and `HALItem` are now included as top-level imports as of v0.3.1 (should be on PyPI now).
